deps/jerry/tests/jerry/uint8array-property-slice.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var null_u8array = new Uint8Array();
+var u8array = new Uint8Array([1, 2, 3, 4]);
+
+var u8arr1 = u8array.slice();
+var u8arr2 = u8array.slice("a", "3");
+var u8arr3 = u8array.slice(-2);
+var u8arr4 = u8array.slice(Infinity, NaN);
+var u8arr5 = u8array.slice(undefined, -3);
+
+
+assert (null_u8array.toString() == '[object Uint8Array]');
+
+assert (u8arr1.length == 4);
```

Comment:
Strict equality should be preferred to loose equality.

Reply:
Yes, I have modified it, thanks :)